osu.Game/Overlays/Settings/SettingsButton.cs

```diff
@@ -20,13 +21,15 @@ public SettingsButton()
 
         public LocalisableString TooltipText { get; set; }
 
+        [Resolved]
+        private LocalisationManager localisation { get; set; }
+
         public override IEnumerable<string> FilterTerms
         {
             get
             {
                 if (TooltipText != default)
-                    // TODO: this won't work as intended once the tooltip text is translated.
-                    return base.FilterTerms.Append(TooltipText.ToString());
+                    return base.FilterTerms.Append(localisation.GetLocalisedBindableString(TooltipText).Value);
```

Comment:
In addition, the base english text should also always be included, as many people who use a non-english language will still search for keywords in english (by habit or preference).